api/jsonapi.go

```diff
@@ -1185,6 +1242,13 @@ func (i *jsonAPIHandler) POSTInventory(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+
+	err = i.node.PublishInventory()
+	if err != nil {
+		ErrorResponse(w, http.StatusInternalServerError, err.Error())
+		return
+	}
```

Comment:
@placer14 Do you have ideas on how to make this work with little time spent? The only other way I could think of was passing an object to every mutating method on InventoryDB which seemed uglier than this route. I don't think there's time for a more elaborate abstraction but if you have one in mind let me know.

Reply:
No concrete ideas. Thinking further, we might allow higher-level abstractions to offer callbacks on the model's lifecycle. This might be a nice way around it. Something like..

`repo.Inventory.AfterChange(func() error)` would offer the behavior injection and then inside/around the setup of the node, once we have all the context required for the update:
```
f := func() error {
  // context managed in one place
}
repo.Inventory.AfterChange(f)
```

Not necessary for this PR though. Just thinking how I might approach this.